ncm-opennebula/src/main/perl/OpenNebula/AII.pm

```diff
@@ -309,9 +319,11 @@ sub aii_install
     my $tree = $config->getElement($path)->getTree();
 
     my $instantiatevm = $tree->{vm};
-    $self->info("Instantiate VM flag is set to: $instantiatevm");
+    my $msg = '';
+    $msg .= "Instantiate VM flag is set to: $instantiatevm. " if defined($instantiatevm);
```

Comment:
same as above only add the flag, not the whole sentence, e.g. 
`$msg .= " instantiate VM flag $instantiatevm";`